fe/fe-core/src/main/java/com/starrocks/sql/optimizer/GroupExpression.java

```diff
@@ -41,6 +41,9 @@
     private final Map<PhysicalPropertySet, Pair<Double, List<PhysicalPropertySet>>> lowestCostTable;
     private final Set<OutputInputProperty> validOutputInputProperties;
     private Map<OutputInputProperty, Integer> propertiesPlanCountMap;
+    // required property by parent -> output property
+    private Map<PhysicalPropertySet, PhysicalPropertySet> outputPropertyMap;
```

Comment:
there are too many maps, set in this class. which are all necessary?

Reply:
add comment